renderdoc/driver/gl/gl_shader_refl.cpp

```diff
@@ -283,7 +283,7 @@ GLuint MakeSeparableShaderProgram(WrappedOpenGL &gl, GLenum type, vector<string>
   gl.glGetProgramiv(sepProg, eGL_LINK_STATUS, &status);
 
   // allow any vertex processing shader to redeclare gl_PerVertex
-  if(status == 0 && type != eGL_FRAGMENT_SHADER && type != eGL_COMPUTE_SHADER)
+  if(!IsGLES && status == 0 && type != eGL_FRAGMENT_SHADER && type != eGL_COMPUTE_SHADER)
```

Comment:
Yes, it is not required to redeclare gl_PerVertex in case of GLES.

There is a paragraph in the GL 4.5 core spec (below 7.4.1 Shader Interface Matching), which is missing from the GL ES spec:

> To use any built-in input or output in the gl_PerVertex block in separable
program objects, shader code must redeclare that block prior to use. A separable
program will fail to link if ...